packages/flutter/test/widgets/view_test.dart

```diff
@@ -450,7 +450,7 @@ void main() {
     expect(children, isNot(contains(rawViewOwner)));
   });
 
-  testWidgetsWithLeakTracking('RenderView does not use size of child if constraints are tight', (WidgetTester tester) async {
+  testWidgets('RenderView does not use size of child if constraints are tight', (WidgetTester tester) async {
```

Comment:
Why no leak tracking?

Reply:
Between reverting and relanding this change the testWidgetsWithLeakTracking method was removed in https://github.com/flutter/flutter/pull/140239. It's functionality is now folded into `testWidgets` directly.